qa/tasks/cephfs/fuse_mount.py

```diff
@@ -144,9 +144,12 @@ def _list_fuse_conns(self):
 
         self.client_remote.run(args=['sudo', 'modprobe', 'fuse'],
                                check_status=False)
-        self.client_remote.run(
-            args=["sudo", "mount", "-t", "fusectl", conn_dir, conn_dir],
-            check_status=False, timeout=(30))
+
+        mounts = self.client_remote.is_mounted(conn_dir)
+        if conn_dir not in mounts:
+            self.client_remote.run(
+                args=["sudo", "mount", "-t", "fusectl", conn_dir, conn_dir],
+                timeout=30)
```

Comment:
since `check_status=False` is removed, you're saying that we should abort if we hit an error? Will this have any impact?

Reply:
The impact is intended. I've mentioned this in detail in commit message.